.circleci/config.yml

```diff
@@ -0,0 +1,174 @@
+version: 2.1
+
+quality_gem_version: &quality_gem_version python-37
+
+orbs:
+  quality: bluelabs/quality@0.0.2
+
+commands:
+  installvenv:
+    description: "Installs dependencies in venv if needed"
+    parameters:
+      python_version:
+        type: string
+    steps:
+      - restore_cache:
+          key: deps-v1-<<parameters.python_version>>-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "setup.py" }}
+      - run:
+          name: Install python deps in venv
+          command: |
+            if [ -f venv/bin/activate ]
+            then
+              . venv/bin/activate
+            else
+              python -m venv venv
+              . venv/bin/activate
+              pip install --upgrade pip
+              pip install --progress-bar=off -r requirements.txt -e .
+            fi
+            # venv/ dir doesn't seem to save enough info to keep the
+            # '-e .' installation.
+            pip install --progress-bar=off -e .
+      - save_cache:
+          key: deps-v1-<<parameters.python_version>>-{{ .Branch }}-{{ checksum "requirements.txt" }}-{{ checksum "setup.py" }}
+          paths:
+            - "venv"
+
+jobs:
+  test:
+    parameters:
+      python_version:
+        type: string
+        description: "Version of python to test against"
+      coverage:
+        type: boolean
+        # The python version can affect the mypy and test coverage in
+        # subtle ways - coverage enforcement should be set only on the
+        # version which matches local development.
+        default: false
+        description: "Enforce coverage not slipping"
+    docker:
+      - image: circleci/python:<<parameters.python_version>>
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            # Bluelabs Butler Key
```

Comment:
Do we need to consider using a different key here? Tbh, haven’t given it a lot of thought, but it’s precisely because I haven’t that I am troubled!

Reply:
Turns out we can do everything we need without this specified at all!  I dropped it.